WcaOnRails/db/seeds/countries.seeds.rb

```diff
@@ -1,207 +1,204 @@
+Country.create(id: "Sri Lanka", name: "Sri Lanka", continentId: "_Asia", iso2: "LK")
+Country.create(id: "Sudan", name: "Sudan", continentId: "_Africa", iso2: "SD")
+Country.create(id: "Suriname", name: "Suriname", continentId: "_South America", iso2: "SR")
+Country.create(id: "Swaziland", name: "Swaziland", continentId: "_Africa", iso2: "SZ")
+Country.create(id: "Sweden", name: "Sweden", continentId: "_Europe", iso2: "SE")
+Country.create(id: "Switzerland", name: "Switzerland", continentId: "_Europe", iso2: "CH")
+Country.create(id: "Syria", name: "Syria", continentId: "_Asia", iso2: "SY")
+Country.create(id: "Taiwan", name: "Taiwan", continentId: "_Asia", iso2: "TW")
+Country.create(id: "Tanzania", name: "Tanzania", continentId: "_Africa", iso2: "TZ")
+Country.create(id: "Thailand", name: "Thailand", continentId: "_Asia", iso2: "TH")
+Country.create(id: "Togo", name: "Togo", continentId: "_Africa", iso2: "TG")
+Country.create(id: "Tonga", name: "Tonga", continentId: "_Oceania", iso2: "TO")
+Country.create(id: "Trinidad and Tobago", name: "Trinidad and Tobago", continentId: "_North America", iso2: "TT")
+Country.create(id: "Tunisia", name: "Tunisia", continentId: "_Africa", iso2: "TN")
+Country.create(id: "Turkey", name: "Turkey", continentId: "_Europe", iso2: "TR")
+Country.create(id: "Turks and Caicos Islands", name: "Turks and Caicos Islands", continentId: "_North America", iso2: "TC")
+Country.create(id: "Tuvalu", name: "Tuvalu", continentId: "_Oceania", iso2: "TV")
+Country.create(id: "Uganda", name: "Uganda", continentId: "_Africa", iso2: "UG")
+Country.create(id: "Ukraine", name: "Ukraine", continentId: "_Europe", iso2: "UA")
+Country.create(id: "United Arab Emirates", name: "United Arab Emirates", continentId: "_Asia", iso2: "AE")
+Country.create(id: "United Kingdom", name: "United Kingdom", continentId: "_Europe", iso2: "GB")
+Country.create(id: "Uruguay", name: "Uruguay", continentId: "_South America", iso2: "UY")
+Country.create(id: "USA", name: "USA", continentId: "_North America", iso2: "US")
+Country.create(id: "Uzbekistan", name: "Uzbekistan", continentId: "_Asia", iso2: "UZ")
+Country.create(id: "Vanuatu", name: "Vanuatu", continentId: "_Oceania", iso2: "VU")
+Country.create(id: "Venezuela", name: "Venezuela", continentId: "_South America", iso2: "VE")
+Country.create(id: "Vietnam", name: "Vietnam", continentId: "_Asia", iso2: "VN")
+Country.create(id: "XA", name: "Multiple Countries (Asia)", continentId: "_Asia", iso2: "XA")
+Country.create(id: "XE", name: "Multiple Countries (Europe)", continentId: "_Europe", iso2: "XE")
+Country.create(id: "XS", name: "Multiple Countries (South America)", continentId: "_South America", iso2: "XS")
+Country.create(id: "Yemen", name: "Yemen", continentId: "_Asia", iso2: "YE")
+Country.create(id: "Zambia", name: "Zambia", continentId: "_Africa", iso2: "ZM")
+Country.create(id: "Zimbabwe", name: "Zimbabwe", continentId: "_Africa", iso2: "ZW")
```

Comment:
Could you please check if this doesn't make test much slower? (Multiple queries.)